x-pack/osquerybeat/include/fields.go

```diff
@@ -19,5 +19,5 @@ func init() {
 // AssetFieldsYml returns asset data.
 // This is the base64 encoded zlib format compressed contents of fields.yml.
 func AssetFieldsYml() string {
```

Comment:
This is also not right. Read my first comment.

Reply:
I feel like too much time has been spent and this submission is of little value. 

Feel sorry, I hope other developers can submit a new PR based on this.